.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I usually add `.editorconfig` to git since it allows the entire team to [share the same settings](https://www.viget.com/articles/use-editorconfig-to-manage-coding-styles-on-team-projects) (if we all have the plugin).